src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/settings/AlertAddressForm.tsx

```diff
@@ -175,27 +180,25 @@ export const AlertAddressForm = (props: Props) => {
             </AlertAddressRadio>
           </AlertAddressContext.Provider>
         )}
-
-        {props.enabledFeatureFlags.includes("MonthlyActivityEmail") &&
-          hasPremium(props.user) && (
-            <ActivateEmailsCheckbox
-              isSelected={activateMonthlyMonitorReport}
-              onChange={handleMonthlyMonitorReportToggle}
-            >
-              <div>
-                <b>
-                  {l10n.getString(
-                    "settings-alert-preferences-allow-monthly-monitor-plus-report-title",
-                  )}
-                </b>
-                <p>
-                  {l10n.getString(
-                    "settings-alert-preferences-allow-monthly-monitor-plus-report-subtitle",
-                  )}
-                </p>
-              </div>
-            </ActivateEmailsCheckbox>
-          )}
+        {props.enabledFeatureFlags.includes("MonthlyActivityEmail") && (
```

Comment:
Great, thanks. And a heads-up: when you merge in the latest `main`, you'll get a merge conflict - that's because the list of feature flags is now an array. So to resolve the conflict, you'll have to add it to the array instead of the type.